chart/docker-auth/README.md

```diff
@@ -76,8 +76,11 @@ The following table lists the configurable parameters of the docker-auth chart a
 | --------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------- |
 | **Secret**                                                             |
-| `secret.data.server.certificate`                                                               | Content of server.pem                                                                                                                                                                                                                                                                         |                        |
-| `secret.data.server.key`                                                        | Content of server.key                                                                                                                                                                                                                                                                                                                           |                           |
+| `secret.data.server.certificate`                                             | Content of server.pem  (mutually exclusive with secretName, keyName, certificateName)                                                                                                                                                                                                                                                                       |                        |
+| `secret.data.server.key`                                                     | Content of server.key  (mutually exclusive with secretName, keyName, certificateName)                                                                                                                                                                                                                                                                                                                        |                           |
+| `secret.secretName`                                                          | The name of the secret containing server key and certificate (mutually exclusive with secret.data.server.key/certificate)         | |
+| `secret.certificateName`                                                     | The name of the server certificate file (mutually exclusive with secret.data.server.key/certificate)                               | |
```

Comment:
Cert-Manager will always use the key `tls.crt` so you should set that as the default for the certificateName